### DISCUSSION TITLE
Mainsail UI

### DISCUSSION THREAD

**Discussion by dweenz** [OrcaSlicer/OrcaSlicer#1169] [Ideas]
I have just started with Orcaslicer and love it however when I use it to connect to my printers, if I do not add the port number :7126, and :7125 after my host ip I cannot send the print to the correct printer. I am running linux on a NUC with 2 printers connected. If I do add the port number I can send prints to each printer properly but cannot use mainsail in the Orcaslicer app as it shows a different ip than the one I have entered but it also shows everything is connected properly just no access to moonraker UI  mainsail. I'm fine with it but if there is something I am missing it would be nice to know. 

![image](https://github.com/SoftFever/OrcaSlicer/assets/123034774/230ff41d-5d9c-45b9-b875-a9119595f11f)

**Comment by AmshaSharam** [comment]
On printer network tab config-> Hostname, ip or url : you put ip: port of printer where files go ex: 192.168.1.45:7125
on "Device ui you put only the ip of interface : ex: 192.168.1.45
![orca_settings_net](https://github.com/SoftFever/OrcaSlicer/assets/97839782/b39ef9ee-f112-490f-8b11-16a699e892fd)

**Reply by dweenz** [reply]
Thanks I did eventually figure it out by trial and error but you are exactly right.

**Reply by Dwall36b** [reply]
I was stumped on this. Thank you for helping me out! I've also shared this to help a few others.
